akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/OptionalSnapshotStoreSpec.scala

```diff
@@ -53,21 +52,18 @@ class OptionalSnapshotStoreSpec extends ScalaTestWithActorTestKit(s"""
 
   import OptionalSnapshotStoreSpec._
 
-  // Needed for the untyped event filter
-  implicit val untyped = system.toUntyped
-
   "Persistence extension" must {
     "initialize properly even in absence of configured snapshot store" in {
-      EventFilter.warning(start = "No default snapshot store configured", occurrences = 1).intercept {
+      LoggingEventFilter.warning(start = "No default snapshot store configured", occurrences = 1).intercept {
         val stateProbe = TestProbe[State]()
         spawn(persistentBehavior(stateProbe))
         stateProbe.expectNoMessage()
       }
     }
 
     "fail if PersistentActor tries to saveSnapshot without snapshot-store available" in {
-      EventFilter.error(pattern = ".*No snapshot store configured.*", occurrences = 1).intercept {
-        EventFilter.warning(pattern = ".*Failed to save snapshot.*", occurrences = 1).intercept {
+      LoggingEventFilter.error(pattern = ".*No snapshot store configured.*", occurrences = 1).intercept {
```

Comment:
If we're going to change this perhaps we could go a bit further - but that's for another PR

Reply:
Not sure what you mean. This specifically tests that the error log is emitted.